test/test_trajectory_sim.py

```diff
@@ -50,7 +50,7 @@ def test_sim_trajectory():
                     np.array(
                         test_trajectory["rotations"][i], dtype=np.float32
                     ),
-                    state.rotation,
+                    np.array([*state.rotation.imag, state.rotation.real]),
```

Comment:
Maybe a typo in `imag`?

Reply:
No typo.  It's short for `imaginary` and returns the coefficients of the `i, j, k` components (the imaginary part) of the quaternion.